### PR TITLE
Add mobcap to scarpet

### PR DESCRIPTION
**Pull request by Ghoulboy78**
running mobcap() will return mobcaps for the dimension,
Somehow IDEA screwed up the imports. Sorry.

### CODE REVIEW

File: src/main/java/carpet/script/CarpetExpression.java [hunk 1]
```diff
@@ -99,11 +92,7 @@
 import net.minecraft.structure.StructurePiece;
 import net.minecraft.structure.StructureStart;
 import net.minecraft.text.LiteralText;
-import net.minecraft.util.ActionResult;
```
Comment (gnembon) [inline src/main/java/carpet/script/CarpetExpression.java:102]: Yeah, fix those, and change in your IDE not to do that again.

Comment (Ghoulboy78) [reply src/main/java/carpet/script/CarpetExpression.java:102]: how to fix the IDE tho?

**Comment by Ghoulboy78** [comment]
Now it returns a list, but the code is kinda messy. It basicaly gives list: `l('MONSTER','3','CREATURE','4')`,blabla
I dunno how to fix that or if I even wanna fix it. Idk man, ur call.

**Comment by gnembon** [comment]
will trash it for now.

Reason is that in 1.16 - its considerably easier to get that information from the game, and 1.16 is right around the corner. Will add that to 1.16 branch instead.

**Comment by Ghoulboy78** [comment]
yeah, true, but you can't get it scarpet can u?

**Comment by Ghoulboy78** [comment]
ah no wait I got ur point. ok

**Comment by gnembon** [comment]
your solution is the same as running `hostile_counts = for(entity_list('*'), _ ~'category'=='monster')`

**Comment by Ghoulboy78** [comment]
oh ofc. That wud have been easier.

**Pull request closed without merging**